src/components/AppStyle.tsx

```diff
@@ -29,10 +29,14 @@ const palette = {
   link: "#0094FF",
   linkDark: "#007FD6",
   warning: "#faad14",
-  successLight: "#b7eb8f",
-  errorLight: "#ffa39e",
-  infoLight: "#91d5ff",
-  warningLight: "#ffe58f",
+  successMedium: "#b7eb8f",
+  successLight: "#f6ffed",
```

Comment:
Something sort of sad to me about a color named `successLight`
*Non blocking comment, just don't want to paint my room with it*